Mail2Bug/WorkItemManagement/TFSWorkItemManager.cs

```diff
@@ -413,6 +474,33 @@ private static void ValidateAndSaveWorkItem(WorkItem workItem)
             workItem.Save();
         }
 
+        private IEnumerable<int> InternalAttachFiles(int workItemId, IEnumerable<string> fileList)
```

Comment:
This function doesn't seem to be used anywhere - is this a remnant from an intermediate version of the code?